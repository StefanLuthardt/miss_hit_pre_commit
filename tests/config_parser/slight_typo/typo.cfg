supresss_rule: "naming_parameters"
