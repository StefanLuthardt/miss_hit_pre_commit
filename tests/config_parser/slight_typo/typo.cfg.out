In typo.cfg, line 1
| supresss_rule: "naming_parameters"
| ^^^^^^^^^^^^^ error: expected valid style configuration name (did you mean suppress_rule?)
typo.cfg: error: config file contains errors
MISS_HIT Debug Summary: 1 file(s) analysed, 2 error(s)
